src/data/abbreviations.js

```diff
@@ -48,6 +48,7 @@ let compact = {
   ],
 
   Place: [
+    'Str',
```

Comment:
yeah, those abbreviations are used in the sentence-splitter, so it doesn't chop-it-up there